pkg/codegen/testing/test/program_driver.go

```diff
@@ -194,12 +190,13 @@ type CheckProgramOutput = func(*testing.T, string, codegen.StringSet)
 type GenProgram = func(program *pcl.Program) (map[string][]byte, hcl.Diagnostics, error)
 
 type ProgramCodegenOptions struct {
-	Language   string
-	Extension  string
-	OutputFile string
-	Check      CheckProgramOutput
-	GenProgram GenProgram
-	TestCases  []ProgramTest
+	Language             string
+	Extension            string
+	OutputFile           string
+	Check                CheckProgramOutput
+	GenProgram           GenProgram
+	TestCases            []ProgramTest
+	AcceptCodegenChanges bool
```

Comment:
It was nice to have as a dev-only thing because GoLand didn't pick up my environment variable `PULUMI_ACCEPT` so  this overrides it, personally I like having this in here instead of env vars. It is false everywhere by default unless you set it somewhere to be `true` while working with codegen tests

Reply:
I'm concerned about having a variable that should only be set to `false`, I think it is confusing. I would at least like a comment explaining why it exists. It would be better if you could corral GoLand into picking up your environmental variable.

Reply:
I've removed `AcceptCodegenChanges` now and will look for a better way to improve the DX when testing